README.md

```diff
@@ -113,6 +113,22 @@ environment and prepared source, compiling any extensions using the
 appropriate override environment settings (refer to
 [customization](docs/customization.md) for details about overrides).
 
+## Using private registries
+
+Fromager uses the [requests](https://requests.readthedocs.io) library
+and `pip` at different points for talking to package registries. Both
+support authenticating to remote servers in various ways. The simplest
+way to integrate the authentication with fromager is to have a
+`~/.netrc` file with a valid entry for the host. For example, to use a
```

Comment:
pip and requests also support `NETRC` env var. Should we document this, too?

Reply:
Makes sense, I'll look at that.

@Gregory-Pereira if we can set a variable, maybe we don't need to move files around in the build job?

Reply:
Updated